modules/generator/processor/spanmetrics/config.go

```diff
@@ -10,49 +10,60 @@ import (
 const (
 	Name = "span-metrics"
 
-	dimService       = "service"
+	dimJob           = "job"
 	dimSpanName      = "span_name"
 	dimSpanKind      = "span_kind"
 	dimStatusCode    = "status_code"
 	dimStatusMessage = "status_message"
+	dimInstance      = "instance"
 )
 
 type Config struct {
 	// Buckets for latency histogram in seconds.
 	HistogramBuckets []float64 `yaml:"histogram_buckets"`
 	// Intrinsic dimensions (labels) added to the metric, that are generated from fixed span
-	// data. The dimensions service, span_name, span_kind, and status_code are enabled by
+	// data. The dimensions job, span_name, span_kind, status_code, and instance are enabled by
 	// default, whereas the dimension status_message must be enabled explicitly.
 	IntrinsicDimensions IntrinsicDimensions `yaml:"intrinsic_dimensions"`
 	// Additional dimensions (labels) to be added to the metric. The dimensions are generated
 	// from span attributes and are created along with the intrinsic dimensions.
 	Dimensions []string `yaml:"dimensions"`
+	// Dimension label mapping to allow the user to rename attributes in their metrics
+	DimensionMappings []DimensionMappings `yaml:"dimension_mappings"`
 
 	// If enabled attribute value will be used for metric calculation
 	SpanMultiplierKey string `yaml:"span_multiplier_key"`
 }
 
 func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet) {
 	cfg.HistogramBuckets = prometheus.ExponentialBuckets(0.002, 2, 14)
-	cfg.IntrinsicDimensions.Service = true
+	cfg.IntrinsicDimensions.Job = true
 	cfg.IntrinsicDimensions.SpanName = true
 	cfg.IntrinsicDimensions.SpanKind = true
 	cfg.IntrinsicDimensions.StatusCode = true
+	cfg.IntrinsicDimensions.Instance = true
+}
+
+type DimensionMappings struct {
+	Name        string   `yaml:"name"`
+	SourceLabel []string `yaml:"source_labels"`
+	Join        string   `yaml:"join"`
 }
 
 type IntrinsicDimensions struct {
-	Service       bool `yaml:"service"`
+	Job           bool `yaml:"job"`
```

Comment:
does this impact existing metrics? we should not be make breaking changes to existing metrics w/o figuring out how we would roll that out to cloud.